### PR TITLE
Shayan/89186/setup playwright

### PR DESCRIPTION
**Pull request by shayan-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Comment by ali-hosseini-deriv** [comment]
i'm wondering isn't it better to move everything to end to end dir and not to include in the root package.json?

File: playwright.config.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    },
+    ignoreHTTPSErrors: true,
+    testIdAttribute: 'data-testid',
+  },
+
+
+  /* Folder for test artifacts such as screenshots, videos, traces, etc. */
+  outputDir: 'test-results/',
+
+  /* Run your local dev server before starting the tests */
```
Comment (ali-hosseini-deriv) [inline playwright.config.ts:53]: is it possible to enable auto start on tests run?

File: end-to-end-tests/onboarding.ts [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+        await this?.page?.evaluate(`window.location.reload();`)
+        const server_url = await this?.page?.evaluate(() => {
+            const result = localStorage.getItem('config.server_url')
+            return Promise.resolve(result)
+        })
+        expect(server_url).toBe(process.env.ENDPOINT!)
+    }
+    async demoWizardHandler() {
+        // await this.page.locator('.static-dashboard-wrapper__header > h2', { hasText: 'CFDs' })
+        // await this.page.locator('.static-dashboard-wrapper__header > h2', {
```
Comment (ali-hosseini-deriv) [inline end-to-end-tests/onboarding.ts:95]: remove if not needed

File: .env.example [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# To have needed env variables you need to make a copy out of this file and rename it to `.env.development`
+# you can change any of values listed here to your desired ones
+# also you can pass any environment variable to this file and have access to them using `process.env.[ENVIRONMENT_VARIABLE_NAME] (eg. process.env.APPID)
+ENDPOINT_PAGE_APP_STORE=false
+ENDPOINT_PAGE_DBOT_DASHBOARD=false
+ENDPOINT_PAGE_DEBUG_SERVICE_WORKER=false
```
[the remark this answers is not in the document]
Comment (shayan-deriv) [reply .env.example:6]: removed

File: .env.example [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+APPID=<YOUR APPID (on local machine it should be `9999`)>
+ENDPOINT=<YOUR DESIRED ENDPOINT (QABOX ADDRESS)>
+APP_URL=<APP URL(FOR LOCAL MACHINES SHOULD BE `http://localhost:8000`>
+TRADING_APP_URL=<TRADING APP URL(FOR LOCAL MACHINES SHOULD BE `https://localhost.binary.sx`>
+QA_EMAIL_INBOX_USER_NAME=<QABOX EVENTS PAGE USERNAME (YOU CAN FIND IN `fe-shared` LASTPASS FOLDER) to sign in on `qaxx.deriv.dev/events`>
+QA_EMAIL_INBOX_PASSWORD = <QABOX EVENTS PAGE PASSWORD (YOU CAN FIND IN `fe-shared` LASTPASS FOLDER) to sign in on `qaxx.deriv.dev/events`>
+ACCOUNT_RESIDENCE=<COUNTRY NAME TO SET RESIDENCY OF ACCOUNT(SHOULD BE `Capitalize` CASE Like `Germany`)>
```
[the remark this answers is not in the document]
Comment (shayan-deriv) [reply .env.example:13]: what do you mean?

File: end-to-end-tests/onboarding.ts [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+
+config({ path: `.env.development` })
+
+export default class OnboardingFlow {
+    readonly page: Page
+    readonly email: string
+    constructor(page: Page) {
+        const randomString = new Date().getTime()
+        this.page = page
+        this.email = `deriv-fe-e2e-${randomString}@deriv.com`
```
[the remark this answers is not in the document]
Comment (shayan-deriv) [reply end-to-end-tests/onboarding.ts:12]: done

File: end-to-end-tests/onboarding.ts (hunk 2 above)
Comment (shayan-deriv) [reply end-to-end-tests/onboarding.ts:95]: done

File: .env.example (hunk 4 above)
[the remark this answers is not in the document]
Comment (shayan-deriv) [reply .env.example:13]: @ali-hosseini-deriv

File: playwright.config.ts (hunk 1 above)
Comment (shayan-deriv) [reply playwright.config.ts:53]: done

**Comment by shayan-deriv** [comment]
> i'm wondering isn't it better to move everything to end to end dir and not to include in the root package.json?

I agree. moved them into a separate path

**Review by habib-deriv — DISMISSED** [review]

**Comment by mohsen-deriv** [comment]
hey man @shayan-deriv :wave: 
do we need this PR ? 
if so, let's move it forward.
if not can you please close this? 
thanks :pray:

**Comment by ali-hosseini-deriv** [comment]
> hey man @shayan-deriv 👋 do we need this PR ? if so, let's move it forward. if not can you please close this? thanks 🙏

hi @mohsen-deriv 
lets discuss in PV about this to see if we can and if we need to have E2E test on deriv-com or not.

**Pull request closed without merging**